src/DGtal/geometry/volumes/distance/ExactPredicateLpPowerSeparableMetric.ih

```diff
@@ -311,7 +311,7 @@ DGtal::ExactPredicateLpPowerSeparableMetric<T,2,P>::binarySearchHidden(const Abs
 								       const Abscissa &/*upper*/) const
 {   
   ASSERT(false && "Not Necessary for l_2");
-  
+  return 0;
```

Comment:
Maybe not the right place to talk about this, but there is a lake of DGtal exceptions, especially with a string member to describe why an exception was raised (e.g. in DGtal::IOException).